src/app/compose/compose.component.html

```diff
@@ -24,13 +24,16 @@
             </button>
             <button *ngIf="!editing" mat-icon-button (click)="editDraft()" matTooltip="Edit draft" id="editDraftIcon">
                 <mat-icon svgIcon="pencil"></mat-icon>
-            </button>                 
+            </button>
+            <button *ngIf="editing" mat-icon-button matTooltip="Save as template" (click)="saveTemplate(true)" id="sendMail">
```

Comment:
The "id" needs changing here - should be unique

Reply:
Good catch, is an id even necessary? I'll have a look.

Reply:
I guess we do need them for the automated tests